pages/resources/lobby.mdx

```diff
@@ -0,0 +1,208 @@
+
+```json
+{
+  "id": "1350184413060661332",
+  "application_id": "891436243903728565",
+  "metadata": {
+    "topic": "balls"
+  },
+  "members": [
+    {
+      "id": "852892297661906993",
+      "metadata": null,
+      "flags": 1
+    }
+  ]
+}
+```
+
+### Lobby Member Object
+
+Represents a member of a lobby.
+
+###### Lobby Member Structure
+
+| Field         | Type                 | Description                                                                       |
+| ------------- | -------------------- | --------------------------------------------------------------------------------- |
+| id ^1^        | snowflake            | The ID of the user                                                                |
+| metadata?     | ?map[string, string] | The metadata of the lobby member (max 25 keys, 1024 characters per key and value) |
+| flags?        | integer              | The [lobby member's flags](#lobby-member-flags)                                   |
+| connected ^2^ | boolean              | Whether the member is connected to a call in lobby                                |
+
+^1^ Not included when returned [over the Gateway](/topics/gateway-events#lobbies).
+
+^2^ Only included in lobby objects returned [over the Gateway](/topics/gateway-events#lobbies).
+
+###### Lobby Member Flags
+
+| Value  | Name           | Description                                       |
+| ------ | -------------- | ------------------------------------------------- |
+| 1 << 0 | CAN_LINK_LOBBY | Lobby member can link a text channel to the lobby |
+
+## Endpoints
+
+<RouteHeader method="POST" url="/lobbies">
+  Create Lobby
+</RouteHeader>
+
+Creates a new lobby. Returns a [lobby](#lobby-object) object on success. Fires a [Lobby Create](/topics/gateway-events#lobby-create) Gateway event.
+
+<Alert type="warn">
+
+This endpoint is not usable by user accounts.
+
+</Alert>
+
+###### JSON Params
+
+| Field                 | Type                                               | Description                                                                                                       |
+| --------------------- | -------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| metadata?             | ?map[string, string]                               | The metadata of the lobby (max 25 keys, 1024 characters per key and value)                                        |
+| members?              | array[[lobby member](#lobby-member-object) object] | The members of the lobby (max 25)                                                                                 |
+| idle_timeout_seconds? | integer                                            | How long to wait (in seconds) before shutting down a lobby after it becomes idle (min 5, max 604800, default 300) |
+
+<RouteHeader method="PUT" url="/lobbies">
+  Join or Create Lobby
+</RouteHeader>
```

Comment:
this (and others) are missing the supportsOAuth2 directive
also if the ep only works w oauth2 (and not with a user token) it needs a warning for that too, like the game relationship endpoints have